cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -113,6 +113,68 @@ func New(prime *primer.Values, args ...string) *CmdTree {
 
 	applyLegacyChildren(stateCmd, globals)
 
+	groups := []captain.CommandGroup{
+		{
+			Message: "Package Management:",
```

Comment:
This isn't localized. And the colon at the end is presentation, not something that we should be concerned with at this level.